src/commands/create/interactive.ts

```diff
@@ -86,6 +93,7 @@ export async function askCreateOptions(
                             name: "Nitro.js",
                             value: "nitrojs",
                         },
+                        { name: "Nuxt", value: "nuxt" },
```

Comment:
The order of these entries is very sensitive, as it guides the user on what we would like them to deploy more on our website (statistically, what is placed on the top, will be more deployed than what is placed on the bottom of this list). For now, we can let it this way because I'm planning to reorder them anyway, but keep in mind for next time you modify user facing behavior.